src/main/scala/mesosphere/marathon/MarathonSchedulerActor.scala

```diff
@@ -416,7 +415,7 @@ class SchedulerActions(
   def stopApp(driver: SchedulerDriver, app: AppDefinition): Future[_] = {
     appRepository.expunge(app.id).map { successes =>
       if (!successes.forall(_ == true)) {
```

Comment:
It is not your code but WTF?? they are always all true but errors are not handled correctly

(Not part of your PR)...